src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -139,12 +143,301 @@ class IndexLogEntryTest extends SparkFunSuite {
             .Columns(Seq("col1"), Seq("col2", "col3")),
           schema.json,
           200)),
-      Content("rootContentPath", Seq()),
+      Content(Directory("rootContentPath")),
       Source(SparkPlan(expectedSourcePlanProperties)),
       Map())
     expected.state = "ACTIVE"
     expected.timestamp = 1578818514080L
 
+    val actual = JsonUtils.fromJson[IndexLogEntry](jsonString)
     assert(actual.equals(expected))
   }
+
+  // CONTENT Tests
+  test("Test files api lists all files from Content object.") {
```

Comment:
Can you remove `// CONTENT Tests` and merge with test description? E.g, `test("Test Content.fromDirectory() for a directory without sub-directories")`

Same for other places.

Reply:
Oh wait. You are testing `Content.files` here right? Then you don't need to depend on `Files.createDirectories` nor `Content.fromDirectory`, but just hand craft `Directory` object?

Reply:
oh yes that is correct. I can do it that way. But i think this one is better because the `Content` object really depends on whether we are building it on windows or linux. (e.g. root in windows is "file:/C:/" whereas on linux it's "file:/". So handcrafted path would not work on build machines (linux) I guess.